tools/runners/Odin.py

```diff
@@ -0,0 +1,16 @@
+from BaseRunner import BaseRunner
+
+
+class Odin(BaseRunner):
+    def __init__(self):
+        super().__init__("odin")
+
+    def run_cb(self, tmp_dir, params):
+
+        self.cmd = ['odin_II', '--permissive', '-o odin.blif', '-V']
+
+        # odin doesn't seem to support include directories
+        # and thus only list of files is provided to it
```

Comment:
What does that mean? What are the implications of that?

Reply:
ODIN II tool doesn't list any way to provide list of include directories in its help message (`odin_II -h`) and this will be a big issue for tests that depend on that, ie. `SweRV` and `Ibex` cores use include directories for their headers